LICENSE

```diff
@@ -1,4 +1,4 @@
-AWS Tools for Microsoft Visual Studio Team Services
+AWS Toolkit for Azure Devops
 
 Copyright (c) 2017 Amazon, Inc
```

Comment:
Update the date. We might want to update all of the dates with this change, in fact.

Reply:
I'll update all the dates with the next change I make (which touches almost all of the source files). Updated these now though.

Reply:
Actually this was the only case of it (other than untouched source files which are fine to leave alone)